packages/cli/configuration/fern/definition/docs.yml

```diff
@@ -34,6 +34,12 @@ types:
       display-name: string
       icon: optional<string>
       slug: optional<string>
+      href:
+        type: optional<string>
+        docs: |
+          If `href` is set, clicking on the tab will redirect to the given URL.
+
+          Tabs with `href` must not have children in the navigation config.
```

Comment:
Are you up to add some docs here: https://buildwithfern.com/learn/docs/config/navigation ?

Reply:
done